CASE-STUDIES-AND-EXPERIENCES.md

```diff
@@ -5,23 +5,16 @@ title: Case Studies and Experiences
 site_side: true
 tags: security-qualitative-metrics
 project: true
-level: 4
 type: documentation
 pitch: The OWASP Security Qualitative Metrics is the most detailed list of metrics which evaluate security level of web projects. It shows the level of coverage of OWASP ASVS. 
 ---
 
+# ![Project Logo](assets/images/logo3_small.png) OWASP Security Qualitative Metrics
 
-
-# ![Project Logo](images/logo3_small.png) Case Studies and Experiences 
-# [![OWASP Flagship](https://img.shields.io/badge/owasp-flagship-blue.svg)](https://owasp.org/projects/)
- [![GitHub license](https://img.shields.io/github/license/Naereen/StrapDown.js.svg)](https://github.com/OWASP/www-project-security-qualitative-metrics/blob/master/LICENSE)
```

Comment:
This was for a completely separate user/project

Reply:
It is ok if you can provide a link for the flagship.svg file.

Reply:
This isn’t a flagship project, in fact it isn’t even an up-to-date project if the content is based on ASVS 3.0